### DISCUSSION TITLE
Oxlint pattern mathcing issue

### DISCUSSION THREAD

**Discussion by saravanakumar2504** [oxc-project/oxc#7840] [General]
Currently i wanted to integrate Oxlint  in my app, and i running command in cli `yarn oxlint -c ./oxlint.json "packages/sub-package/pkg-1/**/*.ts" ` and `oxlint.json` looks like below

```
{
	"$schema": "./node_modules/oxlint/configuration_schema.json",
	"exclude": ["node_modules/", "dist/"],
	"plugins": ["import", "typescript"],
	"overrides": [
		{
			"files": ["packages/sub-package/pkg-1/**/*.ts"],
			"rules": {
				"no-console": 1,
				"no-cycle": 1
			}
		}
	]
}
```
 and its working as expceted. Since i'm have multiple sub packages like `pkg-1, pkg-2... pkg-N` if i update my command and config `yarn oxlint -c ./oxlint.json "packages/sub-package/**/*.ts" ` and in overrides like below its not linting all the packages under `sub-package` folder
 
```
 overrides: [{
     "files": ["packages/sub-package/**/*.ts"],
 }]
```

**Comment by Sysix** [comment]
what OS are you using?
